From b337d7acc5ce47cf7f8d03c321ee7bf5e1c8491a Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Tue, 19 Mar 2019 13:07:18 +0800
Subject: [PATCH 608/706] spi-nor: flexspi: add ls1028 support

Generally speaking, nxp,lx2160a-fspi should not be removed, I think it's
an insane action, so add a new entry for ls1028a.

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/mtd/spi-nor/nxp-flexspi.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/mtd/spi-nor/nxp-flexspi.c b/drivers/mtd/spi-nor/nxp-flexspi.c
index 6aef595c7e01..a587454814db 100644
--- a/drivers/mtd/spi-nor/nxp-flexspi.c
+++ b/drivers/mtd/spi-nor/nxp-flexspi.c
@@ -1161,7 +1161,8 @@ static void nxp_fspi_unprep(struct spi_nor *nor, enum spi_nor_ops ops)
 }
 
 static const struct of_device_id nxp_fspi_dt_ids[] = {
-	{ .compatible = "nxp,lx2160a-fspi", .data = (void *)&ls1028a_data, },
+	{ .compatible = "nxp,lx2160a-fspi", .data = (void *)&lx2160a_data, },
+	{ .compatible = "nxp,ls1028a-fspi", .data = (void *)&ls1028a_data, },
 	{ /* sentinel */ }
 };
 MODULE_DEVICE_TABLE(of, nxp_fspi_dt_ids);
-- 
2.17.1

